common/views/components/ApiToolbar/ApiToolbar.tsx

```diff
@@ -121,17 +121,26 @@ async function createTzitzitWorkLink(
 
 function getAnchorLinkUrls() {
   // This function currently only extracts the ids from h2, h3, and h4 tags
-  const getAllIds = [...document.querySelectorAll('h2, h3, h4')].map(
+  const getAllHeadingIds = [...document.querySelectorAll('h2, h3, h4')].map(
     item => item.id
   );
+  // This function extracts divs that have an id within exhibition in the class name
+  // This is to allow us to grab ids from divs from audio and BSL guides in Exhibition Guides
+  const getDivIds = [...document.querySelectorAll('div[id]')].map(item => {
+    return item.className.match(/exhibition/g) ? item.id : null;
```

Comment:
Is this class just "exhibition" or is it based on styled components having Exhibition somewhere in the name? Is it used to style? (Tried to find it but there's a lot of "exhibition" in the codebase it turns out 😄 )

Reply:
It’s a class that contains the word exhibition, it felt like the easiest way to target only exhibition guide and exhibition styled divs

Reply:
My concern is that I believe classes should be for styling only. If a while from now someone were ever to refactor, it wouldn't necessarily be a reflex to check if renaming a class affects any behavioural aspects. An alternative could be to have an ID on the parent or to use data-attributes?  You know the guides section better than I do though!

Reply:
I see what you're saying. I suppose I'm thinking that this tool is for internal use only, so I wanted to make little to no changes to the frontend code. I know that future when a refactor occurred the result would be: you can't grab anchor links for exhibition guides audio or BSL types using the API Toolbar and that would mean a dev would have to refactor this function when it was reported. I agree it isn't really future proof, I'll have a think

Reply:
Like you said it's internal tooling only! Maybe a data-attribute like `data-toolbar-anchor` on the parent (cause you're grabbing all the headings in a certain area, right?) would keep it minimal? Happy to be a rubber duck if needed, and I'm really not dying on any hills 😄

Reply:
I think it is a good hill to hang out on! I'm going to go with the data-attribute, as it gives future-us a clue of the impact refactoring that section will have, and it isn't trying to do something with a class that isn't at all style related.